.github/workflows/benchmark-pipeline.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           version: v1.30.2
         id: install
-      - run: mkdir ~/.kube && echo ${{ secrets.KUBECONFIG }} > ~/.kube/config
+      - run: mkdir ~/.kube && echo "${{ secrets.KUBECONFIG }}" > ~/.kube/config
```

Comment:
kind error was a red herring due to `kind: Config` in the kubeconfig.

The echo just needs to be quoted to preserve whitespace.